packages/legend-art/style/base/_panel.scss

```diff
@@ -200,6 +250,10 @@
   &__section {
     cursor: default;
     margin: 0.5rem 0 3rem;
+
+    &--padding {
```

Comment:
As mentioned, I think we could create a new component for this purpose instead. If we don't address this, let's keep this comment here.